tests/book.tests.js

```diff
@@ -0,0 +1,123 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const { Book, Genre, Author, Reader } = require('../src/models/index');
+const app = require('../src/app');
+
+describe('/books', () => {
+  let books;
+  beforeEach(async () => {
+    await Book.sequelize.sync({ force: true });
+    await Book.destroy({ where: {} });
+
+    const genre1 = await Genre.create({ genre: 'Dystopian' });
+    const author1 = await Author.create({ author: 'Pierce Brown' });
+    const reader1 = await Reader.create({
+      name: 'Tom',
+      email: 'tom@aol.com',
+      password: 'supersecret12345',
+    });
+    const genre2 = await Genre.create({ genre: 'Sci-Fi' });
+    const author2 = await Author.create({ author: 'Ernest Cline' });
+    const reader2 = await Reader.create({
+      name: 'Amy',
+      email: 'Amy@aol.com',
+      password: 'supersecret12345',
+    });
+    books = await Promise.all([
```

Comment:
Some clear lines in the tes files would be really helpful, everythings very bunched up at the minute and it makes it harder to read than if things were separated a bit 
```
    });

    books = await Promise.all([
```